src/main/java/com/google/finapp/SpannerDaoJDBCImpl.java

```diff
@@ -129,7 +135,10 @@ public void moveAccountBalance(ByteArray fromAccountId, ByteArray toAccountId, B
         }
       }
       if (sourceAmount == null || destAmount == null) {
-        throw new IllegalArgumentException();
+        throw new IllegalArgumentException("Account not found");
```

Comment:
is there a way we could specify which account is not found? maybe by its account id or something? not necessary but could be helpful when testing.

Reply:
+1